src/Sentry.NLog/SentryTarget.cs

```diff
@@ -169,39 +166,48 @@ protected override void Write(LogEventInfo logEvent)
 
                 evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
 
-                if (Options.SendContextPropertiesAsTags)
+                // Always apply any manually configured tags
+                evt.SetTags(GetTags(logEvent));
+
+
+                if (Options.SendEventPropertiesAsData)
```

Comment:
Remove this code. Instead only depend on `IncludeEventProperties` and `contextProps = GetAllProperties(logEvent)`